packages/peregrine/lib/talons/SearchBar/useAutocomplete.js

```diff
@@ -46,12 +44,10 @@ export const useAutocomplete = props => {
     // run the query once on mount, and again whenever state changes
     useEffect(() => {
         if (visible && valid) {
-            setLoading(true);
             runQuery({ variables: { inputText: value } });
-        } else if (!value) {
-            resetState();
         }
-    }, [resetState, runQuery, setLoading, valid, value, visible]);
+        // TODO: resetState ?
```

Comment:
Should still be able to `else if (!value) { resetState() }` here.

Reply:
`resetState` was something we added in `useQueryResult`.

I'm hoping that we don't even need to do that anymore.

Reply:
Ah, I thought it was an external callback. You're right, we don't need that anymore.